packages/web-react/src/components/Tooltip/useFloating.ts

```diff
@@ -30,6 +32,7 @@ type UseTooltipUIProps = {
   tooltipArrowWidth?: number;
   tooltipMaxWidth?: number;
   tooltipPlacement?: Placement;
+  trigger?: ('click' | 'hover' | 'manual')[];
```

Comment:
What about creating a better type and using `TOOLTIP_TRIGGER.HOVER` in the code? You are repeating `click, hover, manual` in a few places again.